app/lib/matador/index.server.ts

```diff
@@ -163,5 +163,5 @@ const getRedisKeys = async (redis: Redis, pattern: string) => {
 };
 
 export const getMatadorVersion = () => {
-  return process.env.MATADOR_VERSION;
+  return process.env.MATADOR_VERSION ?? "Error";
```

Comment:
I think it's better to return an empty string or `"not provided"` i guess

Reply:
👍🏼